sbt-dotty/src/dotty/tools/sbtplugin/DottyPlugin.scala

```diff
@@ -174,7 +179,42 @@ object DottyPlugin extends AutoPlugin {
           scalaVersion.value.split("\\.").take(2).mkString(".")
         else
           scalaBinaryVersion.value
-      }
+      },
+
+      scalaInstance := Def.taskDyn {
```

Comment:
Why is this a `taskDyn`? Is it to only evaluate `fetchArtifactsOf ` if `isDotty` is true?

Reply:
Yes